### ISSUE TITLE
[Android] CRC32C and Installing crcmod

### ISSUE THREAD

**Issue by oldergod** [circleci/circleci-images#100]
TL;DR: `gsutil` requires the _compiled version_ of `crcmod` to work. Now, the not compiled version is installed.

-----------------------------------------------

`gsutil` depends on `crcmod` and seems to require that the latter being in its "compiled version".

You can see the detail here: https://cloud.google.com/storage/docs/gsutil/addlhelp/CRC32CandInstallingcrcmod

At the moment, `crcmod` is indeed being installed but when checked, the installed version wasn't the _compiled version_. This creates error on some `gsutil` commands.

https://github.com/circleci/circleci-images/blob/3ea1b489116c7eab099b6185b574cb33361533de/android/Dockerfile.m4#L25-L26

Once logged onto a CircleCI build with `ssh`, I could rightly installed the _compiled version_ of `crcmod` using the commands Google includes in the above documentation. Namely, installing `crcmod` before installing it: `sudo pip uninstall crcmod`

**Comment by iynere** [comment]
this is done !
